app/views/questions/file_with_help/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :page_title, "File with the help of a tax expert!" %>
+
+<% content_for :form_card do %>
+  <div class="form-card">
+    <h1 class="form-question">
+      <%= content_for(:page_title) %>
+    </h1>
+
+    <p>
+      You selected our full service option. We’ll have you answer a series of questions about your tax situation (to the best of your ability) and have a tax specialist review everything with you.
+    </p>
+
+    <p>
+      This is the right option for you if:
+    </p>
+
+    <ul class="list--checkmark">
+      <li>You need assistance with tax filing</li>
+      <li>Your tax situation is confusing/complicated</li>
+      <li>You’re nervous to file completely on your own</li>
+    </ul>
+    <div class="button--group">
+      <%= link_to feelings_questions_path, class: "button text--centered" do %>
```

Comment:
I was not sure whether to hard code this to the previous page or to literally link back in case this page will be accessed from multiple pages. This keeps people form going to the wrong place if they access this page using the url. As it is now it will need to be changed if we change the flow to this page.

Reply:
I think a `link_to(:back) do` would likely make sense here, essentially just acting close to a browser back button. I've noticed in some future mocks that people may arrive to this page from more than one page.